keyboards/alt67/alt67.h

```diff
@@ -0,0 +1,23 @@
+#ifndef _ALT67_H_
+#define _ALT67_H_
+
+#include "samd51j18a.h"
+#include "config.h"
```

Comment:
This shouldn't be necessary, actually.  The build script adds this by default, IIRC